Source/Diagnostics/ReducedDiags/FieldMomentum.cpp

```diff
@@ -0,0 +1,147 @@
+/* Copyright 2019-2020
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#include "FieldMomentum.H"
+#include "WarpX.H"
+#include "Utils/WarpXConst.H"
+#include "Utils/CoarsenIO.H"
+
+#include <AMReX_REAL.H>
```

Comment:
Some missing includes to increase robustness:
```suggestion
#include <AMReX.H>
#include <AMReX_Geometry.H>
#include <AMReX_MultiFab.H>
#include <AMReX_ParallelDescriptor.H>
#include <AMReX_ParmParse.H>
#include <AMReX_REAL.H>
```